core/lib/libfdt/fdt_ro.c

```diff
@@ -8,6 +8,9 @@
 #include <fdt.h>
 #include <libfdt.h>
 
+/* OP-TEE helper function to leveral cached information from the DT */
```

Comment:
I think we have quite a few copies of this comment already. Can we remove some of them?

Reply:
Another way to less hack into **libfdt** would be to rename the modified libfdt functions and implement them in `core/kernel/dt.c`. For example, regarding `fdt_node_offset_by_phandle()`:
In `core/lib/libfdt/fdt_ro.c` and `core/lib/libfdt/include/libfdt.h`:
```patch
-int fdt_node_offset_by_phandle(const void *fdt, uint32_t phandle)
+int _fdt_node_offset_by_phandle(const void *fdt, uint32_t phandle)
```
and in `core/kernel/dt.c`:
```c
int fdt_node_offset_by_phandle(const void *fdt, uint32_t phandle)
{
	int offset = 0;

	if (fdt_find_cached_node_phandle(fdt, phandle, &offset) == 0)
		return offset;

	return _fdt_node_offset_by_phandle(fdt, phandle)
}
```

Would it be preferable?